pyproject.toml

```diff
@@ -27,10 +27,11 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-nautobot = "^1.4.0"
+nautobot = "^1.5.13"
 diffsync = "^1.6.0"
 Markdown = "!=3.3.5"
 packaging = "^21.3"
+nautobot-capacity-metrics = "^2.0.0"
```

Comment:
What is this needed for?

Reply:
If this is not needed, we need to pull in `prometheus_client` as an explicit dependency.

Reply:
This was added because I thought we need the capacity metrics App to enable metric capability in Nautobot. I believe that has the prometheus_client dependency.

Reply:
Yup - I think it makes more sense pulling in `prometheus_client` directly, although even the `nautobot` dependency should take care of that so it might not make sense to be more specific then necessary here. Still I think we should remove `nautobot-capacity-metrics` as I don't think it provides any value here.

Reply:
Changed to prometheus client as requested.

Reply:
So I'm not sure this is going to work. The capacity-metrics App is needed for the `register_metric_func` function that's used in the `__init__.py` file. I can't find that function in prometheus-client. Should I revert to capacity-metrics?

Reply:
Check [this](https://docs.nautobot.com/projects/core/en/stable/plugins/development/#prometheus-metrics) - specifically the part on `metrics.py` - you don't need register_metric_func anymore.

Reply:
Ah, I was going off [this](https://github.com/nautobot/nautobot-plugin-capacity-metrics#option-2---registry-for-third-party-plugins) in capacity-metrics that said they had to be registered. I'll remove that then.